robot/Cumulus/resources/NPSP.py

```diff
@@ -361,16 +361,21 @@ def fill_bge_form(self, **kwargs):
                 self.salesforce._populate_field(locator, value)
      
          
-    def verify_details_address(self,field,npsp_street, npsp_city, npsp_country):   
+    def verify_address_details(self,field,value,**kwargs):
         """Validates if the details page address field has specified value"""   
         locator= npsp_lex_locators['detail_page']['address'].format(field)
         street, city, country = self.selenium.get_webelements(locator)
-        if street.text ==  npsp_street and city.text == npsp_city and country.text == npsp_country:
-            return "pass"
-        else:
-            return "fail"
-   
-    def validate_checkbox(self,name,checkbox_title):   
+
+        status = None
+        for key, value in kwargs.items():
+            if street.text == kwargs.get("street")  and  city.text == kwargs.get("city") and country.text == kwargs.get("country"):
+                status = "pass"
+            else:
+                status = "fail"
+        if value == "contains":
+            assert status == "pass", "Expected value to be  but found "
```

Comment:
this assert message doesn't give much info, may be u could mention what are the expected and actual values. It will be easy to troubleshoot in case of failure